### DISCUSSION TITLE
Transfer repos and keeping contributions

### DISCUSSION THREAD

**Discussion by eduardolbruno** [community/community#65103] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi guys!
I have 2 github accounts, and need to transfer all the contribution from one account to the other. The main goal is to centralise all my work in just one account, and delete the extra account as I only want to keep one. I think that transferring the repos should be enough but don’t wanna try it and loose all my old work. 

Can someone point me in the right direction? (Simple step by step would be enough)

Also, what would happen with my contributions? Can I have all the contributions listed in one account after transferring the repos?

Thx in advance, happy coding! 🫶🏻

**Comment by arafajs** [comment]
You cannot. the only transfer can be done for organizations, not for personal accounts.

**Reply by eduardolbruno** [reply]
Oh I see, and if I transfer each repo one by one, that could work? I saw the option to transfer repos but want to know if I can keep my contributions when I transfer them to the other account.

**Reply by arafajs** [reply, marked as answer]
Yes, you can keep your contributions when you transfer repositories from one GitHub account to another.

**Reply by eduardolbruno** [reply]
Perfect, thanks 🙌🏻

**Comment by 0dayhunter** [comment]
Of course, I'll provide you with simplified answers to your questions:

**1. How to transfer repositories from one GitHub account to another?**
   - Log in to the source account.
   - Go to the repository.
   - Click on "Settings" (gear icon).
   - Choose "Transfer" and follow the instructions to the destination account.

**2. What happens with my contributions after transferring repositories?**
   - New contributions will be associated with the destination account.
   - Old contributions stay with the source account, and there's no automatic way to move them.

**3. Can I have all the contributions listed in one account after transferring the repos?**
   - No, historical contributions remain with the original account.

**4. What about my GitHub accounts after the transfer?**
   - You can keep the accounts or archive/delete the source account.

Remember, transferring repositories doesn't transfer historical contributions. New contributions will be attributed to the destination account, but you can't combine old contributions from the source account with the new one.

**Reply by eduardolbruno** [reply]
Thanks mate! Will try this and see how it goes.
